### ISSUE TITLE
[Idea] Retain downloaded updates and choose from them.

### ISSUE THREAD

**Issue by Neyunse** [renpy/renpy#4237]
I was thinking that it would be interesting to have a system like Unity Hub has. That keeps the versions of the engine in a folder and so choose which version of the engine to create the project in. This way it would not be necessary to re-download the update and overwrite the renpy files, for example you could create in the update screen buttons that refer to different versions of the engine; and when you download a particular one in the settings screen you can decide which version of renpy to use for the projects. This would also help developers who had problems with the latest versions to select a version already installed and not have to download again an update. They would simply decide the version of RenPy they already have downloaded.

Example:
![imagen](https://user-images.githubusercontent.com/36079551/211627542-7697b230-4ca6-4133-a68a-0b73fb6a49f8.png)

Visual example:

![imagen](https://user-images.githubusercontent.com/36079551/211630900-a59817c6-8ce3-40ba-b8f7-b9181160594b.png)

![imagen](https://user-images.githubusercontent.com/36079551/211632936-a26fd8bf-4025-496d-88ea-04d237b617a4.png)
this example was made with: https://www.renpy.org/doc/html/screens.html#nearrect

And the versions could be downloaded in the update folder and renpy could just copy and overwrite the already downloaded files keeping the update folder or could use a folder called "version" and store here all the already downloaded versions.

ex: ['[path]/renpy/version/renpy-8.0.3', '[path]/renpy/version/renpy-7.4.4',  '[path]/renpy/version/renpy-pre-7.4.3']

in short, I think this would help to ease the movement between old and new versions without having to re-update/download a new version of renpy or download separate engine versions for testing. 

Perhaps it is currently a difficult subject to consider, or not interesting at all at this time. So I leave this as an idea in case it is interesting or if it is decided to be done in the future.

**Comment by renpytom** [comment]
Please don't pseudo-tag issues. 

I don't think this is something I'd want to implement - I don't want to encourage people to use obsolete versions.
